### ISSUE TITLE
The accuracy on cifar is very low

### ISSUE THREAD

**Issue by guoguangchao** [AminParvaneh/alpha_mix_active_learning#4]
Thanks for your great work, I train on cifar100 according to the instructions in readme,But I got a very low accuracy,I use the following command to train:
python main.py \
        --data_name CIFAR100 --data_dir data --log_dir log \
        --n_init_lb 100 --n_query 100 --n_round 10 --learning_rate 0.001 --n_epoch 1000 --model mlp \
        --strategy AlphaMixSampling --alpha_opt
The accuracy is 0.0907
![mix](https://user-images.githubusercontent.com/5878481/195812526-7643dcf0-73cd-4398-8ad3-3c22e7dd06c5.png)
What parameters do I need to change? thanks

**Comment by AminParvaneh** [comment]
The reported results for CIFAR100 are with a budget of size 1000 and a pre-trained Small ViT model. Please find the exact set of hyper-parameters for each experiment in Table 3 of supplements.

**Comment by kanzaali** [comment]
Hi Amin,
Thank you so much for sharing your work. 
Did you use Adam optimizer for bigger datasets, such as CIFAR10, CIFAR100, and MiniIMageNet?
I have also reproduced with CIFAR10 with ResNet18 and the accuracy is very low. 
I think these default settings of your coding are showing accuracy for 1000 labeled samples only, am I getting correct?

**Comment by AminParvaneh** [comment]
We used Adam as the optimizer in all the experiments. Please adjust the default parameters based on Table 3 in the supplements. Also, note that the reported results in the paper are the average of 3 experiments with different random seeds, so you should be able to repeat the results with more or less similar values.

**Comment by kanzaali** [comment]
Thank you so much for the explanation. 
I think you are referring to Table 1 in the supplements as there is no Table 3.

**Comment by AminParvaneh** [comment]
Yes, I am referring to Table 3 in the ArXiv version and Table 1 of the Appendix in the CVPR version.
